app/views/sephora_style_guide/home/index.slim

```diff
@@ -1,4 +1,5 @@
 main role='main'
+  = render 'sephora_style_guide/shared/svgs'
```

Comment:
Let me add the dependency for `inline_svg` to the project.

After that could you remove all this stuff and use it to render the svg? We are not going to use svg sprites any more.